src/coalesce-vue-vuetify3/src/components/input/c-datetime-picker.vue

```diff
@@ -562,6 +553,13 @@ function emitInput(value: Date | null) {
       value = props.max;
     }
 
+    if (!isDateAllowed(value)) {
```

Comment:
I tried to combine this logic to reduce duplication, it seems to be working the same - but I'm putting a note so it's double-checked.